tensorboard/backend/application.py

```diff
@@ -849,3 +876,31 @@ def make_plugin_loader(plugin_spec):
         if issubclass(plugin_spec, base_plugin.TBPlugin):
             return base_plugin.BasicLoader(plugin_spec)
     raise TypeError("Not a TBLoader or TBPlugin subclass: %r" % (plugin_spec,))
+
+
+class ExperimentalPluginLoader(base_plugin.TBLoader):
```

Comment:
Do you have a recommendation about where this class could live? I've put it here just to get the ball rolling but suspect it would be preferable to move elsewhere.

Reply:
Great. Left it here.

Reply:
FWIW I've been trying to reduce what's going on in application.py (cf. #2573) so if you don't mind I'd have a slight preference for a separate module in backend/.

Reply:
Done. It also means I can make it visible to tensorboard:internal, which I think we want for now.